docs/build/p2p-node/index.md

```diff
@@ -161,109 +104,95 @@ $ subkey --sr25519 generate
 
-- **Mainnet**
+#### Network Configurations
 
-Use network `-n mainnet`
+[//]: # (TODO: Update POD link)
 
----
+Besides `mainnet`, Centrifuge has support for the `catalyst` test network. The network configuration for the different testnets is also part of the [code base](https://github.com/centrifuge/go-centrifuge/blob/master/build/configs/default_config.yaml). This enables the client user to run on top of them with minimum configuration needed. Please find the most important information summarized below:
 
-**If you like to run the Centrifuge node with your own Ethereum or Centrifuge Ehain nod,e please replace the Infura urls with your own `Geth` and `centrifuge-chain` Node-URL.**
+##### Catalyst
 
----
+Use network `-n catalyst`.
 
-### Look up your Centrifuge ID
+This network is a testnet that operates over a Proof-of-Stake in the Centrifuge Chain.
 
-A participant of the Centrifuge P2P network will be identified by an identity (Centrifuge ID) within the network. The `createconfig` command automatically creates an identity and the key pairs.
+- Client: parity
+- Purpose: Testnet
+- Bootstrap Nodes:
 
-To look up your Centrifuge ID via Terminal use:
+```
+/ip4/35.242.230.116/tcp/38202/ipfs/12D3KooWSbxbKCbZh9JVtsQyVGdTPra4RpSA4tbvs6an11jwGA2z
+```
 
-```bash
-$ cat /<PATH-TO-CONFIG-DIR>/config.yaml | grep -i 'identityid' | awk '{print $2}'
+```
+/ip4/35.234.72.127/tcp/38202/ipfs/12D3KooWQm2cSmrEiaSMV4gUv7WGhpgRwo8woFSsHhZGbGi3aA8x
 ```
 
-### Running the Centrifuge node after creating the config.yaml
+##### Mainnet
 
-Before running your Centrifuge Node, you need to add your Ethereum key and password as environment variables. The following bash commands show how to set the variables based on the default ethereum key location in macOS for a sample key.
+Use network `-n mainnet`.
 
-<!-- `CENT_ETHEREUM_ACCOUNTS_MAIN_KEY=$HOME/Library/Ethereum/keystore/UTC--2019-0UTC--2019-mm-dd` -->
+This network is the production network that operates over the main Proof-of-Stake in the Centrifuge Chain.
 
-```bash
-CENT_ETHEREUM_ACCOUNTS_MAIN_KEY=$(cat $HOME/Library/Ethereum/keystore/UTC--2019-04-15T14-43-41.293727000Z--75aecbd0aa7f34207132d686d2a9e470fba2e6e4)
-CENT_CENTCHAIN_ACCOUNT_SECRET=<YOUR_CENTCHAIN_SECRET>
-```
+- Client: parity
+- Purpose: Mainnet
+- Bootstrap Nodes:
 
-```bash
-CENT_ETHEREUM_ACCOUNTS_MAIN_PASSWORD=<YOUR_PASSWORD>
 ```
-
-Afterwards, you can run the Centrifuge Node using the `config.yaml` file you created:
-
-```bash
-$ centrifuge run -c /<PATH-TO-CONFIG-DIR>/config.yaml
+/ip4/35.198.122.117/tcp/38202/ipfs/12D3KooWAg3EcAtrYVCxwz6k6sT9iZSRztz9D7yUzpBPREFzrUoV
 ```
 
-Replace the `PATH-TO-CONFIG-DIR` with the location of the `config.yaml` file.
+```
+/ip4/35.242.221.111/tcp/38202/ipfs/ 12D3KooWKGwixXenuXAVqkJKmnHSAJDjzf7eGMo6troigZxm7A5R
+```
 
-## Centrifuge Configuration
+#### Changing the default configuration
 
-### Changing the default configuration
+[//]: # (TODO: Update POD link)
```

Comment:
Is this needed before merging?

Reply:
Yes, I added this not to forget about these links before changing this PR into ready for review.